javascript/code-challenges/challenges-01.test.js

```diff
@@ -136,7 +132,17 @@ This function should use forEach to populate your grocery list based on the stor
 ------------------------------------------------------------------------------------------------ */
 
 const createList = (availableItems) => {
-  // Solution code here...
+  const newList = [];
+
+  availableItems.forEach(item => {
+
```

Comment:
Here is another example of when this spacing could be removed (line 138 and 142). Spacing is great when starting a new code block of course and overall yours was just fine:)